types/jasmine/index.d.ts

```diff
@@ -15,6 +15,7 @@
 //                 Stephen Farrar <https://github.com/stephenfarrar>
 //                 Mochamad Arfin <https://github.com/ndunks>
 //                 Alex Povar <https://github.com/zvirja>
+//                 Spacemonaut <https://github.com/spacemonaut>
```

Comment:
Notice, you should add your name here only if you want to become a maintainer of this project and always participate in future project's life. You will be notified each time a PR is created about jasmine, so it's quite a lot.

Otherwise, you could make changes without adding your name here, as it's not mandatory 😉